src/cameras/index.jsx

```diff
@@ -4,9 +4,11 @@ import { Grid } from '@material-ui/core'
 import { Content } from 'cameras/content'
 import { useDefaultContext, Context } from 'cameras/context'
 import { Sidebar } from 'cameras/sidebar'
+import { Behaviour } from 'cameras/behaviour'
 
 export const Cameras = () => <Context.Provider value={useDefaultContext()}>
   <Grid container>
+    <Behaviour />
```

Comment:
It's ridiculous that I am asking for this, but let's *not* have the `<Behaviour/>` component be a child of this grid container, as it doesn't have to 😄

Reply:
Done